src/components/App/App.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
can we just apply ts-ignore to specific lines instead

Reply:
done!